### PR TITLE
Add alerts and descriptions for Reg Only path to Health Care Application

### PR DESCRIPTION
**Pull request by longmd**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

This PR creates new, reusable content components for the Registration Only path work being completed on the Health Care Application.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#81103

## Acceptance criteria

 - Components are available to drop into future PRs related to this effort

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution

### CODE REVIEW

File: src/applications/hca/tests/unit/components/FormAlerts/AuthenticationShortFormAlert.unit.spec.js
Comment (longmd) [inline src/applications/hca/tests/unit/components/FormAlerts/AuthenticationShortFormAlert.unit.spec.js:1]: Fixing the implementation on this from a previous PR to match the desired pattern for reuse of selectors the selectors object.

File: src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js
Comment (longmd) [inline src/applications/hca/tests/unit/components/FormPages/VeteranInformation.unit.spec.js:1]: Fixing the implementation on this from a previous PR to match the desired pattern for reuse of selectors the selectors object.

File: src/applications/hca/tests/unit/components/VeteranInformation/AuthProfileInformation.unit.spec.js
Comment (longmd) [inline src/applications/hca/tests/unit/components/VeteranInformation/AuthProfileInformation.unit.spec.js:1]: Fixing the implementation on this from a previous PR to match the desired pattern for reuse of selectors the selectors object.

File: src/applications/hca/tests/unit/components/VeteranInformation/GuestVerifiedInformation.unit.spec.js
Comment (longmd) [inline src/applications/hca/tests/unit/components/VeteranInformation/GuestVerifiedInformation.unit.spec.js:1]: Fixing the implementation on this from a previous PR to match the desired pattern for reuse of selectors the selectors object.

File: src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js
Comment (longmd) [inline src/applications/hca/tests/unit/utils/helpers/general.unit.spec.js:1]: Fixing the implementation on this from a previous PR to match the desired pattern for reuse of selectors the selectors object.

**Review by coope93 — APPROVED** [review]

File: src/applications/hca/components/FormAlerts/RegistrationOnlyAlert.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import propTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import recordEvent from 'platform/monitoring/record-event';
+import { APP_URLS } from '../../utils/constants';
+
+const RegistrationOnlyAlert = ({ headingLevel = 3 }) => {
+  const { currentlyLoggedIn } = useSelector(state => state.user.login);
+  const subHeadingLevel = parseInt(headingLevel, 10) + 1;
+  const H = `h${headingLevel}`;
```
[the remark this answers is not in the document]
Comment (longmd) [reply src/applications/hca/components/FormAlerts/RegistrationOnlyAlert.jsx:11]: No, those aren't invalid. Just copied the pattern that I've seen used in quite a few other places, as you mentioned you have seen. I can adjust, if you'd like.

**Pull request merged**